### DISCUSSION TITLE
Qval meaning

### DISCUSSION THREAD

**Discussion by kuleshan** [jackbibby1/SCPA#48] [Q&A] [answered]
Hello Jack!
Thank you very much for such an excellent tool!
I am currently applying it for my datasets, but the meaning of qval still remains unclear to me.
You have already mentioned in one of the previous discussions that qval is transformed adjusted p value, but could you tell a bit about how is it calculated?
Me and my research group are now trying to understand which pathways should we consider as meaningful and knowing this would really help us.

Thank you in advance!

**Comment by jackbibby1** [comment, marked as answer]
Hi,

Hope it has been useful. The qval is a directly related to the pval because it's a basic transformation of the adjusted pval. It's calculated by: $\sqrt{-log(adj.pval)}$. We just include it as an intuitive way for ranking pathways by degree of distributional change. So for your interpretation, you can say that the larger the qval, the larger the distributional change of the pathway i.e. the greater the pathway difference.

So when you're deciding on your pathway importance, I would follow a typical interpretation of statistical output i.e. considering pathways with higher qvals to be more relevant/differentially regulated than those with lower qvals.

Jack

**Reply by kuleshan** [reply]
Thank you very much for your swift reply!
That will help us a lot :)
